src/EnergyPlus/SurfaceGeometry.cc

```diff
@@ -496,103 +467,102 @@ namespace SurfaceGeometry {
 
         for (int ZoneNum = 1; ZoneNum <= state.dataGlobal->NumOfZones; ++ZoneNum) {
```

Comment:
If you are not using `ZoneNum` anywhere (I don't see it) can just make this a range loop.